layers/+completion/auto-completion/packages.el

```diff
@@ -130,6 +131,13 @@
                                             "company-statistics-cache.el"))
       (add-hook 'company-mode-hook 'company-statistics-mode))))
 
+(defun auto-completion/init-company-tabnine ()
+  (use-package company-tabnine
+    :if auto-completion-enable-tabnine
+    :defer t
+    :config
+    (setq company-tabnine-max-num-results 3)))
```

Comment:
This seems unnecessary, unless the defaults are horrible I think they should be left as-is. And if the defaults do need adjusting I think that magic number `3` should instead be a variable, one that is used to limit other completion sources to behave consistently.